fhservices.cfg.j2

```diff
@@ -57,26 +57,26 @@ define host {
        host_name {{ rhmap_router_dns }}
 }
 
-{% for fhservice in fh_services_ping %}
+{% for fhservice in fh_services if 'ping' in fhservice.get("checks") %}
```

Comment:
Where is the code that processes the template?
Is the template processed in run time or build time?